src/cmd/config.go

```diff
@@ -75,20 +93,22 @@ func init() {
 }
 
 func readConfig() []byte {
-	var err error
-	var res []byte
-
-	switch cfgFile {
-	case ".":
-		res, err = os.ReadFile("./opslevel-k8s.yaml")
```

Comment:
This situation should not be possible

1. You cannot provide `.` as a CLI argument (unless you do some weird trickery. It's not a good practice.)
2. This is never set to `.` by default - it's set to `./opslevel-k8s.yaml` [so this code is misleading.](https://github.com/OpsLevel/kubectl-opslevel/blob/c3a97ad6af097f2f82765a7f79965e4b2570e6ca/src/cmd/root.go#L45)